### DISCUSSION TITLE
Estilo musical que escuta quando está desenvolvendo?

### DISCUSSION THREAD

**Discussion by wellingtonpgp** [frontendbr/forum#145] [Discussão]
Hey Devs, gostaria de saber qual estilo musical que os senhores mais gosta, tipo banda, gênero e "etc"  :)

**Comment by afonsopacifer** [comment]
J-Pop / J-Rock ou netflix no segundo monitor <3

**Comment by wilkerviana** [comment]
Deep House / Disclosure \o/

**Comment by yurimatheus** [comment]
A depender do dia e do projeto vou de Arrocha a Remix... sou muito eclético.

Mas, geralmente estou escutando Hardwell ou Calvin Harris.

**Comment by woliveiras** [comment]
Cara, falar pro ce: Antigamente eu só ouvia Rock, Metal e Hip Hop... Hoje em dia eu me forço a ouvir de tudo pra estimular a cabeça. Da uma olhada [nesse artigo](http://www.megacurioso.com.br/musica-e-danca/42628-aprenda-a-usar-a-musica-para-aumentar-a-sua-eficiencia-no-trabalho.htm).

**Comment by felipefialho** [comment]
Cara, eu sou super eclético.

Tenho uma tatuagem do Pink Floyd no braço (em vias de fazer uma do Bowie agora :smile:), porém posso ir de Rock N' Roll a Samba Raiz. De MPB a música clássica. De Country a Sertanejo anos 90, rs.

Mas particularmente na última semana, passei boa parte dela ouvindo umas seleções de músicas clássicas no Spotify e curti bastante.

**Comment by brunoravanhani** [comment]
Pra mim depende do dia, de vez em quando umas coisas tipo Maroon 5, outro dia Johnny Cash, outro Vivaldi, outro dia uns Rap nacional, Samba de raiz, moda de viola caipira,  e de vez em quando barulho de chuva no http://www.noisli.com/ rsrsrsrs, acho que o legal é mudar e ouvir coisas novas.

**Comment by lagden** [comment]
Esse negócio de música é doido...
Eu coloco o som e foco na programação, então a música some, ou seja, somente o inconsciente está escutando, de repente a playlist acaba e penso: carai, já tocou tudo!!!

**Comment by felipefialho** [comment]
Exatamente @lagden, as vezes nem percebo que to escutando música, haha.

**Comment by vitorbritto** [comment]
Para desenvolvimento eu escuto Death Metal, algo como Six Feet Under, Deicide, Vomitory, Dissection, Death, WhiteChapel, TBDM e afins. 

Se for para planejar ou testar, eu escuto música clássica, Frank Sinatra, The Doors, Led Zeppelin, Pink Floyd ou algo mais **leve**.

**Comment by evansantos** [comment]
Eu quando estou codando, costumo colocar aquela playlist marota de jogos/filmes e as vezes música clássica minimalista

**Comment by heliomsolivas** [comment]
Eu fico no Deep House normalmente, mas dependendo do dia pode rolar até sertanejo/funk/metalcore/hc e por ai vai haha

**Comment by CallMeFabioo** [comment]
Durante o desenvolvimento é basicamente Death Metal, Black Metal, Deathcore, etc.
Mas dependendo do dia e do humor, acaba rolando uns instrumental, soundtrack, clássica, samba rock.

**Comment by rickbenetti** [comment]
Aqui o melhor pra concentração é Jazz e Blues, segue link das playlists:

https://open.spotify.com/user/spotify/playlist/5O2ERf8kAYARVVdfCKZ9G7

https://open.spotify.com/user/spotify/playlist/0BTnoKdfv5Y60EK5o5bGaq

**Comment by danilovaz** [comment]
Minha playlist de [foco no front](https://open.spotify.com/user/12151266913/playlist/5zTBcg2woVRhwDCkCmew4w) e a playlist de [inspiração](https://open.spotify.com/user/12151266913/playlist/2YNhEm1QpaIsJbFqY5WkIG)

**Comment by lucasmahle** [comment]
Quando preciso criar: Música clássica;
Quando preciso programar (modo normal): Eletro House, Deep House;
Quando preciso programar (modo HARD!!!): Death Metal, Black Metal, Deathcore etc;
Quando estou mais de boa: Rock nacional, samba, MPB, antigas etc;

**Comment by lucasmahle** [comment]
Ah, não pode esquecer de fazer Air Drum kkk

**Comment by evansantos** [comment]
@lucasmahle  quem nunca... 

![playing-drums-sourpatchk-Ucm82kAjN6W6A](https://camo.githubusercontent.com/4698c3ebd4620e9af7fd8c236e15d8e88125af04/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f55636d38326b416a4e365736412f67697068792e676966)

**Comment by evansantos** [comment]
@brunoravanhani não conhecia o [noisli](http://www.noisli.com/), só conhecia o [coffitivity](https://coffitivity.com) e o [rainy mood](http://www.rainymood.com/)

**Comment by lucasmahle** [comment]
@evansantos As vezes sai um Headbanging também kk
![Headbanging](http://49.media.tumblr.com/56416fe9e3acf25ffbb1be8375904bea/tumblr_n3djmrNVkp1qimi3zo1_500.gif)

**Comment by maxrandrade** [comment]
Cara, enquanto desenvolvo eu gosto muito de ouvir sons mais acústicos. Me dá uma certa leveza nos pensamentos. Quando tava trampando num projeto no final de Janeiro, acho que tive quase 1.000 execuções de Jason Mraz em umas 2 semanas hahahaha.

Além disso, acho muito prazeroso ouvir novos álbuns ou artistas. Eu tendo a gostar mais de músicas que eu conheço enquanto realizo alguma outra atividade.

**Comment by Marcelosilva10** [comment]
Quanto estou em um clima Hardcore um bug eu escuto Enya, quando tá de boa ai ficar na conta de Nickelback, U2, Soja...

**Comment by vitor-mendrone-deel** [comment]
Gosto de codar com Rock 'n Roll mais cadenciado, Doom Metal ou Folk (Norte americano e nórdico) 

No geral o que não falta na playlist pra codar é: 

Woods Of Ypres (Doom Metal)
Solstafir (Difícil descrever, letras em Islandês com temática pagã) 
Eivor (Uma mina das Ilhas Faroé misturando Folk nórdico com outros sons mais modernos)
Wardruna (Folk Nórdico com temática pagã)
The White Buffalo (Folk americano, rola muito em Sons of Anarchy)
The Forest Rangers (Folk Rock americano)
Alcest (Blackgaze em francês)

São musicas mais arrastadas ou mais suaves mas ainda assim pesadas (em sua maioria), me ajudam bastante a manter a concentração.

**Comment by DominguesM** [comment]
Escuto muito k-pop e trap, mas principalmente rap.

**Comment by evansantos** [comment]
Tava no medium e achei esse post... é meio relacionado ao assunto =)

[How Music Affects Your Productivity by Gregory Ciotti](https://medium.com/life-learning/how-music-affects-your-productivity-42a6dfa6fdfe#.hu60x5w3k)

**Comment by felipefialho** [comment]
Pode crê @evansantos, eu li hoje. Puta post!

**Comment by tcelestino** [comment]
Vou do rap, ao rock (metalcore) e música baiana (não é só axé, viu cambada! hahaha)

**Comment by felipefialho** [comment]
Pô, mas música baiana eu penso em Raul Seixas, Dorival Caymmi e Caetano Veloso :yum:

**Comment by Jandersolutions** [comment]
Eu curto trabalhar ouvindo um bom Smooth Jazz,musica Clássica, e quando a coisa aperta...um playlist de New Age.

**Comment by tcelestino** [comment]
@LFeh mas tem outras bandas de "rock alternativo" e de reggae. Não sei se você acompanhou, mas na edição anterior do Superstar teve uma banda baiana lá. Scambo.

**Comment by bernardodiasc** [comment]
Acho que trabalho na internet pq sempre foi fonte interminável de música, nem da pra mencionar minhas favoritas aqui... então vo mandar só a que to ouvindo agora mesmo.

https://www.youtube.com/watch?v=Irz9PNHqt-Y

**Comment by barbier** [comment]
Normalmente eu [sigo esse infográfico](http://wp_media-prod.s3.amazonaws.com/app/uploads/2013/09/sonos-infographic-working-jams-what-music-to-listen-to-on-the-job.jpg)

![image](https://cloud.githubusercontent.com/assets/3603793/15158179/ff5b33e0-16c4-11e6-97f4-b593a36cbe0c.png)

**Comment by felipefialho** [comment]
Bem legal @barbier!

**Comment by barbier** [comment]
Sendo um pouco mais específico:
Eu ouço o que me dá na telha. Eu não consigo me forçar a nada, então quando eu quero ouvir Folk, eu boto José González, quando quero ouvir Celta, ouço Omnia ou Loreenna Mckennitt, quando quero algo próximo de música clássica, vou para temas de filmes ou Two steps from hell....
E quando estou cansado do trabalho e querendo codar em casa, normalmente ouço Atari Teenage Riot e Blood Stain Child para sair da apatia que acontece com frequência comigo...
Uma outra recomendação minha é a trilha sonora de Hearthstone, na verdade eu recomendo trilhas sonoras de jogos em geral para programar, já que de acordo com especialistas™, esse tipo de música serve para deixar o jogador motivado sem deixá-lo desatento.

**Comment by bomffas** [comment]
Geralmente um bom Power Metal.
- Angra
- Helloween
- Kamelot
- Blid Guardian

**Comment by guilhermealveslopes** [comment]
Gosto de música que 'me empolgue'. Chego a sincronizar o audio com o code kkk
Teclado vira instrumento

Braza é massa, Big Up. Sticky FIngers etc
https://www.youtube.com/watch?v=_K4iTacYanI

**Comment by lagden** [comment]
Tinha um rapaz que trampava comigo que escutava Pitty!! Só programa bugs!! 🤣

**Comment by gamarsh94** [comment]
Musicas agitadas muito top 
Veja esse 👇

https://youtu.be/jPEPffsh1WI

**Comment by brunopulis** [comment]
Bom vou tentar pontuar, ouço muita música instrumental e metalcore me ajuda a concentrar bastante.
